api/src/main/java/org/openmrs/module/reporting/report/definition/service/ReportDefinitionServiceImpl.java

```diff
@@ -128,7 +128,21 @@ public void purgeDefinition(ReportDefinition definition) {
 		
 		getService().purgeDefinition(definition);
 	}
-	
+
+	/**
+	 * @see ReportDefinitionService#purgeDefinition(String)
+	 * @param uuid the uuid of the report definition to remove
```

Comment:
You do not need the above second line because the @see annotation has it all.

Reply:
Well spotted! 🍰